mux_test.go

```diff
@@ -245,6 +246,19 @@ func TestDeleteStd(t *testing.T) {
 	require.Equal(t, w.Body.String(), "test successful")
 }
 
```

Comment:
Thank you for this tests ! More tests that cover more cases would be wonderful

- [ ] with groups
- [ ] nested groups